index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Sajjda Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <section>
+    <header id="main-header">
+      <div class="container">
+          <h2 id="logo">Sajjad</h2>
+          <img src="images/menu-icon.png" id="menu-icon" alt="menu icon">
+        </div>
+    </header>
+    <article>
+      <div id="primary-text">I’m Sajjad Glad to see you!</div>
+      <div id="secondary-text">I’m a software developer! I can help you build a product , feature or website Look
+        through some of my work and experience! If you like what you see and have a project you need coded, don’t
+        hestiate to contact me.</div>
+      <div id="connect">LET'S CONNECT</div>
+      <div id="socialmedia">
+        <a href="https://twitter.com/sajjadrajaee">
+          <img src="images/social/twitter.svg" alt="social icons">
+        </a>
+        <a href="https://linkedin.com/in/sajjadrajaee">
+          <img src="images/social/Linkedin.svg" alt="social icons">
+        </a>
+        <a href="#">
+          <img src="images/social/M.svg" alt="social icons">
+        </a>
+        <a href="https://github.com/sajjadrajaee">
+          <img src="images/social/Github.svg" alt="social icons">
+        </a>
+        <a href="#">
+          <img src="images/social/Vector.svg" alt="social icons">
+        </a>
+      </div>
+    </article>
+  </section>
```

Comment:
## OPTIONAL

- [ ] KIndly make sure also you stick to the design by using the same font-size, background-positon margin, and padding as the template is defined. Check this :point_down: 

Your design | the template|
---|--
![image](https://user-images.githubusercontent.com/65068771/164861673-6f570394-3d00-4e88-ab68-d39f9445f9e1.png) | ![image](https://user-images.githubusercontent.com/65068771/164861658-1035212e-0de2-4fc7-aacf-19f59a4ce390.png)